aptos-move/move-examples/token_objects/token_lockup/sources/token_lockup.move

```diff
@@ -0,0 +1,261 @@
+module token_objects::token_lockup {
+   use std::signer;
+   use std::option;
+   use std::error;
+   use std::string::{Self, String};
+   use std::object::{Self, Object, TransferRef, ConstructorRef};
+   use std::timestamp;
+   use aptos_token_objects::royalty::{Royalty};
+   use aptos_token_objects::token::{Self, Token};
+   use aptos_token_objects::collection;
+
+   #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+   struct LockupConfig has key {
+      last_transfer: u64,
+      transfer_ref: TransferRef,
+   }
+
+   /// The owner of the token has not owned it for long enough.
+   const ETOKEN_IN_LOCKUP: u64 = 0;
+   /// The owner must own the token to transfer it
+   const ENOT_TOKEN_OWNER: u64 = 1;
+
+   const COLLECTION_NAME: vector<u8> = b"Rickety Raccoons";
```

Comment:
Lol Rickety Racoons

Reply:
'm having fun with a different alliterative collection name for each of the examples